pkg/cosign/tlog.go

```diff
@@ -91,7 +91,7 @@ func intotoEntry(ctx context.Context, signature, pubKey []byte) (models.Proposed
 // There are two Env variable that can be used to override this behaviour:
 // SIGSTORE_REKOR_PUBLIC_KEY - If specified, location of the file that contains
 // the Rekor Public Key on local filesystem
-func GetRekorPubs(ctx context.Context, _ *client.Rekor) (map[string]RekorPubKey, error) {
+func GetRekorPubs(ctx context.Context) (map[string]RekorPubKey, error) {
```

Comment:
This is the only reason it's a breaking change, right? I'll bring this up in the community meeting tomorrow but I'd love an official decision that Cosign's library API isn't subject to semver guarantees, then we wouldn't have to wait for v2 I think

Reply:
There's also `VerifyBundle` and `VerifyTLogEntry`: I really did want to make this change at some point (I can leave as TODOs though): it's really odd that those verification func signatures currently require an online rekor client: they only require a trusted rekor key.

Reply:
Given the state of the library part of cosign, I'm not overly concerned with getting this in now vs 2.0.